modular_citadel/code/modules/arousal/organs/womb.dm

```diff
@@ -1,41 +1,11 @@
 /obj/item/organ/genital/womb
-	name 			= "womb"
-	desc 			= "A female reproductive organ."
-	icon			= 'modular_citadel/icons/obj/genitals/vagina.dmi'
-	icon_state 		= "womb"
-	zone 			= "groin"
-	slot 			= "womb"
-	internal 		= TRUE
-	fluid_id 		= "femcum"
-	producing		= TRUE
-
-/obj/item/organ/genital/womb/Initialize()
-	. = ..()
-	reagents.add_reagent(fluid_id, fluid_max_volume)
-
-/obj/item/organ/genital/womb/on_life()
-	if(QDELETED(src))
-		return
-	if(reagents && producing)
-		generate_femcum()
-
-/obj/item/organ/genital/womb/proc/generate_femcum()
-	reagents.maximum_volume = fluid_max_volume
-	update_link()
-	if(!linked_organ)
-		return FALSE
-	reagents.isolate_reagent(fluid_id)//remove old reagents if it changed and just clean up generally
-	reagents.add_reagent(fluid_id, (fluid_mult * fluid_rate))//generate the cum
-
-/obj/item/organ/genital/womb/update_link()
-	if(owner)
-		linked_organ = (owner.getorganslot("vagina"))
-		if(linked_organ)
-			linked_organ.linked_organ = src
-	else
-		if(linked_organ)
-			linked_organ.linked_organ = null
-		linked_organ = null
-
-/obj/item/organ/genital/womb/Destroy()
-	return ..()
+	name = "womb"
+	desc = "A female reproductive organ."
+	icon = 'modular_citadel/icons/obj/genitals/vagina.dmi'
+	icon_state = "womb"
+	zone = BODY_ZONE_PRECISE_GROIN
+	slot = ORGAN_SLOT_WOMB
+	internal = TRUE
+	fluid_id = "femcum"
+	producing = TRUE
```

Comment:
Why is it the womb that produces fluid anyway? Shouldn't it be handled by the vagina?

Reply:
Well... It's been this way even before this PR, the only thing I did here was remove some stupid tabbing.
My main focus is cleaning up the code, not improving the biology artistic license of these features, though I'll take in consideration this peeve.